MVP_Clicker/Assets/Project/Configs/Resources/Bonuses/Bonuses(Types)/Sword.asset

```diff
@@ -0,0 +1,18 @@
+%YAML 1.1
```

Comment:
It bad way to call path like this "Bonuses/Bonuses(Types)"
Instead you can change to "Bonuses/BonusesTypes" or something like this